docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -10,245 +10,242 @@
 title: LoopBack 4
 url: index.html
 children:
+  - title: 'Getting started'
+    url: Getting-started.html
+    output: 'web, pdf'
 
-- title: 'Getting started'
-  url: Getting-started.html
-  output: 'web, pdf'
-
-- title: 'Examples and tutorials'
-  url: Examples-and-tutorials.html
-  output: 'web, pdf'
-  children:
-
-  - title: 'Todo Tutorial'
-    url: todo-tutorial.html
+  - title: 'Examples and tutorials'
+    url: Examples-and-tutorials.html
     output: 'web, pdf'
     children:
-
-    - title: 'Create your app scaffolding'
-      url: todo-tutorial-scaffolding.html
-      output: 'web, pdf'
-
-    - title: 'Adding legacy juggler'
-      url: todo-tutorial-juggler.html
-      output: 'web, pdf'
-
-    - title: 'Add your Todo model'
-      url: todo-tutorial-model.html
-      output: 'web, pdf'
-
-    - title: 'Add a Datasource'
-      url: todo-tutorial-datasource.html
-      output: 'web, pdf'
-
-    - title: 'Add a Repository'
-      url: todo-tutorial-repository.html
-      output: 'web, pdf'
-
-    - title: 'Add a Controller'
-      url: todo-tutorial-controller.html
-      output: 'web, pdf'
-
-    - title: 'Putting it all together'
-      url: todo-tutorial-putting-it-together.html
-      output: 'web, pdf'
-
-    - title: 'Bonus: Integrate with a geo-coding service'
-      url: todo-tutorial-geocoding-service.html
-      output: 'web, pdf'
-
-  - title: 'TodoList Tutorial'
-    url: todo-list-tutorial.html
+      - title: 'SOAP Web Service Tutorial'
+        url: soap-calculator-tutorial.html
+        output: 'web, pdf'
+        children:
+          - title: 'SOAP Web Service Overview'
+            url: soap-calculator-tutorial-web-service-overview.html
+            output: 'web, pdf'
+    children:
```

Comment:
I'm seeing a duplicate `children` here. The proposed change looks like:
```yaml
title: LoopBack 4
url: index.html
children:
  - title: 'Getting started'
    url: Getting-started.html
    output: 'web, pdf'

  - title: 'Examples and tutorials'
    url: Examples-and-tutorials.html
    output: 'web, pdf'
    children:
      - title: 'SOAP Web Service Tutorial'
        url: soap-calculator-tutorial.html
        output: 'web, pdf'
        children:
          - title: 'SOAP Web Service Overview'
            url: soap-calculator-tutorial-web-service-overview.html
            output: 'web, pdf'
    children: (Is this duplicate?)
      - title: 'Todo Tutorial'
        url: todo-tutorial.html
        output: 'web, pdf'
```

Reply:
Thank you !, I removed it.